src/Aquifer.Common/Services/Caching/CachingLanguageService.cs

```diff
@@ -61,7 +61,7 @@ public async Task<IReadOnlyDictionary<string, int>> GetLanguageIdByCodeMapAsync(
 
                 return
                     (LanguageCodeById,
-                    LanguageIdByCode: LanguageCodeById.ToDictionary(li => li.Value, li => li.Key));
+                    LanguageIdByCode: LanguageCodeById.ToDictionary(li => li.Value, li => li.Key, StringComparer.OrdinalIgnoreCase));
```

Comment:
This will allow passing language codes like "hiN" and still getting back results.